src/__snapshots__/core.test.ts.snap

```diff
@@ -104,7 +104,6 @@ declare module \\"nextjs-routes\\" {
     | { pathname: \\"/[foo]/baz\\"; query: Query<{ foo: string }> }
     | { pathname: \\"/_debug\\"; query?: Query | undefined }
     | { pathname: \\"/_debug/health-check\\"; query?: Query | undefined }
-    | { pathname: \\"/_error\\"; query?: Query | undefined }
```

Comment:
The `_error` page shouldn't be included, since it's marked as a non-routable page [here](https://github.com/tatethurston/nextjs-routes/blob/3eee33e62b8c8981e54ebf9eb301aafdb243f5cf/src/core.ts#L5)